test/wallet/signerTest.ts

```diff
@@ -0,0 +1,276 @@
+import { assert } from "chai";
```

Comment:
This file should just be `test/wallet/signer.ts` - it's already in the test folder, so you know it's a test file